### ISSUE TITLE
Refresh Interval

### ISSUE THREAD

**Issue by eliasah** [jprante/elasticsearch-jdbc#355]
I have recently updated my elasticsearch cluster from 1.1.2 -> 1.3.4 along with it's correspondent version of the river.

I use the river to pull data from a MySQL database to the cluster and I've noticed that the refresh_interval is being disabled by default. 

In my case, I run a script that adds the rivers to the cluster then queries the cluster to know if the indexation is done using a delta method.

By delta method, I mean that I compute the difference between document counts in the index within a specific period of time. By disabling the refresh_interval, sometimes due to network performance issues,
the cluster doesn't get data on time and thus by using the delta method, the script thinks that the indexing is done and deleted the river.

To remedy this I force the refresh interval "by hand" inside my script and I believe that this is not a very clean way to do it even thought it works.

In conclusion, I believe that it would be useful to be able to set the interval within the jdbc river parameters.

So what do you think?
